README.md

```diff
@@ -1,2 +1,49 @@
+[![Build Status](https://travis-ci.org/ucsc-cgp/bouncer.svg?branch=master)](https://travis-ci.org/ucsc-cgp/bouncer)
+
 # bouncer
 The whitelist checker for authentication with CGP HCA Data Store
+
+## setup
+
+### regular
+Either add `cgp-bouncer` to your project requirements or
+```
+pip install cgp-bouncer
+```
+in a Python 3 virtualenv.
+
+### for development
+Instead of the steps above clone the repo, `cd` into the repo, and run
+```
+pip install -e .
+```
+
+To test, run
+```
+python -m unittest -v test.py
+```
+## how to use
+Using is simple!
+
+Here's an example
+
+```python
+>>> from bouncer import Bouncer
+>>> b = Bouncer('commons/dev/whitelist')
+>>> b.is_authorized('jrbrenna@ucsc.edu')
+True
+>>> b.is_authorized('evil.gnomes@ucsc.edu')
+False
+```
+
+This checks the AWS Secret Keeper called `commons/dev/whitelist` to see
+if the users `jrbrenna@ucsc.edu` and `evil.gnomes@ucsc.edu` are in the
```

Comment:
I think it would be good to edit this so you don't expose your personal email address unnecessarily on a public site.

Reply:
Good point. Although my email is already exposed in many other places.